app/src/main/java/org/mozilla/fenix/library/bookmarks/BookmarkFragment.kt

```diff
@@ -199,11 +200,13 @@ class BookmarkFragment : LibraryPageFragment<BookmarkNode>(), BackHandler {
                 metrics?.track(Event.OpenedBookmarksInPrivateTabs)
                 true
             }
-            R.id.edit_bookmark_multi_select -> {
+            R.id.share_bookmark_multi_select -> {
                 val bookmark = bookmarkStore.state.mode.selectedItems.first()
                 navigate(
-                    BookmarkFragmentDirections.actionBookmarkFragmentToBookmarkEditFragment(
-                        bookmark.guid
+                    BookmarkFragmentDirections.actionBookmarkFragmentToShareFragment(
+                        url = bookmark.url,
+                        title = bookmark.title,
+                        tabs = arrayOf(ShareTab(bookmark.url!!, bookmark.title!!))
```

Comment:
`url.orEmpty()` and `title.orEmpty()` might be better so that we don't crash if its null.